scan.go

```diff
@@ -150,6 +150,10 @@ func ReadRR(r io.Reader, file string) (RR, error) {
 // The text "; this is comment" is returned from Comment. Comments inside
 // the RR are returned concatenated along with the RR. Comments on a line
 // by themselves are discarded.
+//
+// The parser will not syntax check anything, e.g. illegal base64 in RRSIGs will
```

Comment:
I think we do check somethings sometimes, so perhaps this would be better worded something like ‘Callers should not assume the output is...’.